scripts/basic-workflow-tests.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -euf
+set -xeuf
```

Comment:
`-x` should probably go away once you're done testing?

Reply:
The `-x` is incredibly useful for seeing the commands that actually get run and debugging. I couldn't have made this PR without it. I broke this out in a separate commit (https://github.com/instructlab/instructlab/pull/1860/commits/f9c13c3f19635ebd9eaad9a1f108990b06d5e588) for the sake of proposing the keep it permanently or cutting it from the PR entirely.

Since the log output from CI is usually only viewed when failures occur or for debugging, I think this change has quite a bit of benefit.

Reply:
I'm not opposed to having `-x`.